config/development.js

```diff
@@ -1,6 +0,0 @@
-module.exports = {
-
-	title: 'TinyUrl (development)',
```

Comment:
bad idea like this, create a new key with environment saying dev or prod
don't concatenate string TinyUrl with development or prod like this